src/wagtailmarkdown/edit_handlers.py

```diff
@@ -1,6 +1,12 @@
 import warnings
 
-from wagtail.admin.edit_handlers import FieldPanel
+from wagtail import VERSION as WAGTAIL_VERSION
+
+
+if WAGTAIL_VERSION >= (3, 0):
+    from wagtail.admin.panels import FieldPanel
+else:
+    from wagtail.admin.edit_handlers import FieldPanel
```

Comment:
we might as well drop support for Wagtail < 4.1

Reply:
We were talking about this today in our meeting. Do you mean we should remove these conditionals and also no longer test versions <4.1 in tox and/or CI

It's probably a more general question as well, aimed at the other packages we'll be updating in the coming weeks 😄 
Thanks

Reply:
Now that 2.15 is no longer supported (https://endoflife.date/wagtail), my preference is to drop pre 4.1 logic and support. So this means dropping conditionals, tests on CI etc. I would wager we should even make the lower Wagtail version 4.1 too


Now, this is my preference on the packages I maintain. Different packages will have different preferences.

Reply:
Also, https://github.com/wagtail/wagtail/wiki/Python-Package-Maintenance-Guidelines#setuppy--setupcfg--pyprojecttoml

> Support for EOL versions of Python, Django and Wagtail should be removed from the earliest minor release of the package following the moment the dependency went EOL. This does not apply if provision is made to maintain support for the EOL dependency.

Reply:
Excellent ^^ thank you.

Reply:
I have removed all Wagtail v3 version checks